Circular_Queue/Circular_Queue.cpp

```diff
@@ -0,0 +1,151 @@
+/* 
+Circular Queue is based on FIFO (First In First Out) principle and the last position 
+is connected back to the first position to make a circle.
+*/
+
+#include <iostream>
+using namespace std;
+
+int queue[5];
+int front = -1, rear = -1, size = 5;
+
+void insert (int value);
+void deletequeue();
+void display();
+
+int main()
+{
+    int choice, value;
+    cout << "1.Insert\n";
```

Comment:
Please remove spaces around ``<<`` and ``>>`` in ``cout`` and ``cin`` statements, they are unnecessary and the compiler simply ignores them.